Managers/UTMAppleVirtualMachine.swift

```diff
@@ -323,6 +323,8 @@ import Virtualization
                 }
                 fsConfig.share = self?.makeDirectoryShare(from: newShares)
             }
+
+            try appleConfig.resetDriveSnapShot()
```

Comment:
Any reason why you didn't put this outside of the `if #available(macOS 12, *) {` block?

It should theoretically work for Linux VMs using Apple Virtualization on macOS 11 as well.

Reply:
no reason, haha - will shift it out